index.html

```diff
@@ -167,6 +167,35 @@ <h5 class="donation__bank-account">Name of the bank / Type of bank account</h5>
         </div>
     </section>
 </main>
+<footer class="footer">
+<div class="footer-container">
+    <div class="footer-container__box">
+    <div class="footer-container__contacts">
+        <h3 class="footer__title">For questions and suggestions</h3>
+        <div class="footer__connected">
+            <img src="src/img/mail.svg" alt="Photo">
+            <a class="footer__connected-text">email@shelter.com</a>
+        </div>
+        <div class="footer__connected">
+            <img src="src/img/phone.svg" alt="Photo">
+            <a class="footer__connected-text">+13 674 567 75 54</a>

```

Comment:
add href
look and add another types of links
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a